README.md

```diff
@@ -152,6 +152,85 @@ $
 $ docker run -p 3000:3000 -p 35729:35729 -v /Users/mdl/workspace/mean-stack/mean/public:/home/mean/public -v /Users/mdl/workspace/mean-stack/mean/app:/home/mean/app --link db:db_1 mean
 ```
 
+## Deploying To Cloud Foundry
```

Comment:
I think up until line 188 the info is ok, but beyond it's quite specific to the CF configuration and that's not the place to put it. I suggest that we try one of:
1. We create a Wiki entry and refer to it
2. We create documentation for it on the 0.4.0 gh-pages branch and refer to it.

Also please place the CF README section in the bottom part and definitely not before the core MEANJS items like Getting Started area.